.vscode/sudo_gdb.sh

```diff
@@ -0,0 +1,2 @@
+#!/bin/bash
+sudo gdb "$@"
```

Comment:
Is this going to work on Windows too?

Reply:
This is used for linux port and we currently only have linux port. Are you asking about wls support ? If, yes, it will work.

Reply:
Got it. I thought it's used for device debugging.